nixos/doc/manual/contributing-to-this-manual.chapter.md

```diff
@@ -11,3 +11,25 @@ $ nix-build nixos/release.nix -A manual.x86_64-linux
 ```
 
 If the build succeeds, the manual will be in `./result/share/doc/nixos/index.html`.
+
+**Contributing to the man pages**
+
+The man pages are written in [DocBook] which is XML.
+
+To see what your edits look like:
+
+```ShellSession
+$ cd /path/to/nixpkgs
+$ nix-build nixos/release.nix -A manpages.x86_64-linux
+```
+
+You can then read the man page you edited by running
+
+```ShellSession
+$ man --manpath=result/share/man nixos-rebuild # Replace nixos-rebuild with the command whose manual you edited
+```
+
+If you're on a different architecture that's supported by nixOS (check nixos/release.nix) then replace `x86_64-linux` by the architecture.
```

Comment:
```suggestion
If you're on a different architecture that's supported by NixOS (check nixos/release.nix) then replace `x86_64-linux` with the architecture.
```

Reply:
I actually had to look this up and learned something new again: 

 - replace with = active voice
 - replace by = passive voice

Will modify

Reply:
Yeah, go figure

Reply:
Done